src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
```

Comment:
Having `React` in scope of a component is not needed anymore.

```suggestion
import { useState } from "react";
```

Reply:
It can be removed in other places as well.